docs/discover/search.asciidoc

```diff
@@ -36,28 +47,30 @@ status codes, you could enter `status:[400 TO 499]`.
 codes and have an extension of `php` or `html`, you could enter `status:[400 TO
 499] AND (extension:php OR extension:html)`.
 
-NOTE: These examples use the Lucene query syntax. You can also submit queries
-using the Elasticsearch Query DSL. For examples, see
-{ref}/query-dsl-query-string-query.html#query-string-syntax[query string syntax]
-in the Elasticsearch Reference.
+For more detailed information about the Lucene query syntax, see the
+{ref}/query-dsl-query-string-query.html#query-string-syntax[Query String Query]
+docs.
 
-[float]
-[[save-search]]
-=== Saving a Search
+NOTE: These examples use the Lucene query syntax. When lucene is selected as your
+query language you can also submit queries using the {ref}/query-dsl.html[Elasticsearch Query DSL].
+
+include::kuery.asciidoc[]
+
+[[save-load-search]]
+=== Saving and Loading Searches
```

Comment:
Since it's called "opening" down below, might make sense to use the same terminology here.